app/src/main/java/camp/nextstep/edu/github/GlobalApplication.kt

```diff
@@ -0,0 +1,8 @@
+package camp.nextstep.edu.github
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class GlobalApplication: Application() {
+}
```

Comment:
```suggestion
class GlobalApplication: Application()
```
불필요한 중괄호는 제거해보면 어떨까요?